Client/Frontend/Widgets/ActionOverlayTableViewController.swift

```diff
@@ -73,13 +71,30 @@ class ActionOverlayTableViewController: UIViewController, UITableViewDelegate, U
         tableView.cellLayoutMarginsFollowReadableWidth = false
         tableView.accessibilityIdentifier = "Context Menu"
 
+        let width = min(self.view.frame.size.width, ActionOverlayTableViewUX.MaxWidth) - (ActionOverlayTableViewUX.Padding * 2)
+
         tableView.snp.makeConstraints { make in
             make.center.equalTo(self.view)
-            make.width.equalTo(290)
+            make.width.equalTo(width)
             setHeightConstraint(make)
         }
     }
 
+    private func applyBackgroundBlur() {
+        let appDelegate = UIApplication.shared.delegate as! AppDelegate
+        if let screenshot = appDelegate.window?.screenshot() {
+            let blurredImage = screenshot.applyBlur(withRadius: 5,
```

Comment:
No, Sadly the UIVisualEffectView isnt very customizable. I cant change the blur radius for a softer blur. UIBlurView has these properties but they are private 😠 

I'll fix the rotation issues.